manageiq-operator/api/v1alpha1/helpers/miq-components/kafka.go

```diff
@@ -162,8 +162,17 @@ func KafkaService(cr *miqv1alpha1.ManageIQ, scheme *runtime.Scheme) (*corev1.Ser
 		if len(service.Spec.Ports) == 0 {
 			service.Spec.Ports = append(service.Spec.Ports, corev1.ServicePort{})
 		}
-		service.Spec.Ports[0].Name = "kafka"
-		service.Spec.Ports[0].Port = 9092
+
+		service.Spec.Ports = []corev1.ServicePort{
```

Comment:
I thikn there are issues with recreating the ports _inside_ the func.  @bdunne?

Reply:
I think this is okay since nothing else should be editing the ports